test/integration_tests/test_models.py

```diff
@@ -1,67 +1,48 @@
 import torch
-import torchtext
+from torchtext.models import (
+    XLMR_BASE_ENCODER,
+    XLMR_LARGE_ENCODER,
+    ROBERTA_BASE_ENCODER,
+    ROBERTA_LARGE_ENCODER,
+)
 
 from ..common.assets import get_asset_path
+from ..common.parameterized_utils import nested_params
 from ..common.torchtext_test_case import TorchtextTestCase
 
 
 class TestModels(TorchtextTestCase):
-    def test_roberta_base(self):
-        asset_path = get_asset_path("roberta.base.output.pt")
-        test_text = "Roberta base Model Comparison"
-
-        roberta_base = torchtext.models.ROBERTA_BASE_ENCODER
-        transform = roberta_base.transform()
-        model = roberta_base.get_model()
+    @nested_params(
+        [
+            ("xlmr.base.output.pt", "XLMR base Model Comparison", XLMR_BASE_ENCODER),
+            ("xlmr.large.output.pt", "XLMR base Model Comparison", XLMR_LARGE_ENCODER),
+            (
+                "roberta.base.output.pt",
+                "Roberta base Model Comparison",
+                ROBERTA_BASE_ENCODER,
+            ),
+            (
+                "roberta.large.output.pt",
+                "Roberta base Model Comparison",
+                ROBERTA_LARGE_ENCODER,
+            ),
+        ],
+        [True, False],
+    )
+    def test_model(self, model_args, is_jit):
```

Comment:
nit: This test is rather self-explanatory, so I think it's okay, but I recommend writing a short docstrings for tests so that future maintainers can tell what was my *intention* and see if the implementation makes sense.

Writing docstring helps defining what consists of a correct test, what's the definition, when there is no intuitive test criteria for a new feature.

Reply:
Thanks for the feedback. Just added a short docstring to describing what's being tested